src/content-type/content-type.controller.ts

```diff
@@ -11,12 +19,41 @@ export class ContentTypeController {
   }
 
   @Post('collections/:contentType')
-  createOne(@Query('contentType') contentType: string): Promise<any> {
-    return this.contentTypeService.create(contentType);
+  async createOne(@Param('contentType') contentType: string): Promise<any> {
+    try {
+      const createdType = await this.contentTypeService.create(contentType);
+      return { name: createdType };
+    } catch (err) {
+      if (err.code === 'EEXIST') {
+        throw new HttpException(
+          {
+            status: HttpStatus.CONFLICT,
+            error: 'Collection already exists',
+          },
+          HttpStatus.CONFLICT,
+        );
+      } else {
+        throw err;
```

Comment:
I hope re-throwing an error doesn't kill the server, what http error code does nest respond in this case?